tests/v2/validation/certrotation/cert_rotation.go

```diff
@@ -2,14 +2,23 @@ package certrotation
 
 import (
 	"context"
+	"errors"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
 
+	"github.com/rancher/norman/types"
 	apiv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
 	rkev1 "github.com/rancher/rancher/pkg/apis/rke.cattle.io/v1"
 	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
```

Comment:
There seems to be a large use of management/v3 here. **IF** possible could you convert these to v1 equivalents. If its not possible feel free to ignore this.

Reply:
unfortunately a lot of the rke1 stuff is only in management, so there isn't much that could move over. i.e. all the actions are only v3 right now.